module_27_Python/2_my_math_module.py

```diff
@@ -0,0 +1,57 @@
+"""
+Задача 2. Математический модуль
+Вася использует в своей программе очень много различных математических вычислений,
+связанных с фигурами.
+Например, нахождение их площадей или периметров.
+Поэтому, чтобы не захламлять код огромным количеством функций,
+он решил выделить для них отдельный класс, подключить как модуль и использовать
+по аналогии с модулем math.
+
+Реализуйте класс MyMath, состоящий как минимум из следующих методов
+(можете бонусом добавить и другие методы):
+- вычисление длины окружности,
+- вычисление площади окружности,
+- вычисление объёма куба,
+- вычисление площади поверхности сферы.
+
+Пример основного кода:
+res_1 = MyMath.circle_len(radius=5)
+res_2 = MyMath.circle_sq(radius=6)
+print(res_1)
+print(res_2)
+
+Результат:
+- 31.41592653589793
+- 113.09733552923255
+"""
+
+
+class MyMath:
+    NUMBER_PI = 3.14159265358979323846
+
+    @classmethod
+    def get_circle_length(cls, radius: float) -> float:
```

Comment:
Since it is not a member method of a class, but rather a namespaced function, it is not a getter per se
So, remove pls get_ prefix